app/components/Button.tsx

```diff
@@ -2,14 +2,15 @@ import { PropsWithChildren, ButtonHTMLAttributes } from "react";
 import { tv } from "tailwind-variants";
 
 type ButtonProps = {
-  variant: "primary";
+  variant: "primary" | "disabled";
```

Comment:
disabled should not be a variant, rather a state of the button by the already-included `disabled` prop